### DISCUSSION TITLE
How can I apply styles to Popup?

### DISCUSSION THREAD

**Discussion by Luminisc** [CommunityToolkit/Maui#1318] [Q&A] [answered]
I'm trying to add styles for popup in common `Styles.xaml` to change popup background color (`Color` property), but It doesn't work.
I'm trying this:
```
<Style TargetType="mtk:Popup">
    <Setter Property="Color" Value="{AppThemeBinding Light={StaticResource White}, Dark={StaticResource Gray950}}" />
</Style>
```
But popup is still have generic grey color.

My workaround for now is to copy this value inside popup:
```
<mtk:Popup
    x:Class="..."
    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    xmlns:mtk="clr-namespace:CommunityToolkit.Maui.Views;assembly=CommunityToolkit.Maui"
    Color="{AppThemeBinding Light={StaticResource White},
                            Dark={StaticResource Gray950}}">
```
But I want to add more popups, and avoid this copypasta

**Comment by TheCodeTraveler** [comment, marked as answer]
Popup now supports Styles in v6.1.0

https://learn.microsoft.com/dotnet/communitytoolkit/maui/views/popup#styling
